tests/main/apt-hooks/task.yaml

```diff
@@ -39,6 +39,10 @@ execute: |
             echo "Store is reporting 429 (too many requests), skipping the test"
             exit 0
         fi
+        if "$TESTSTOOLS"/journal-state get-log -u snapd | MATCH "Catalog refresh failed: cannot sections"; then
```

Comment:
This will need updating once #10457 lands.

Reply:
right, thankd for the review